dashboard/src/components/TreeListingPage/TreeListingPage.tsx

```diff
@@ -35,6 +60,52 @@ const TreeListingPage = ({ inputFilter }: ITreeListingPage): JSX.Element => {
     enabled: fastStatus === 'success' && !!fastData,
   });
 
+  function validateString(val: string): boolean {
+    const convertedNumber = parseInt(val);
+    return !Number.isNaN(convertedNumber) && convertedNumber > 0;
+  }
+
```

Comment:
since this is a pure function you can leave outside of the component so we don't have to worry about memoization or keeping an stable reference